tests/class-php-typography-test.php

```diff
@@ -933,8 +933,8 @@ public function test_smart_quotes_off( $html, $result ) {
 	 * @uses PHP_Typography\Text_Parser\Token
 	 */
 	public function test_smart_quotes_french_should_not_apply() {
-		$html   = 'attributs <code>role="group"</code> et <code>aria-labelledby</code>';
-		$result = 'attributs <code>role="group"</code> et <code>aria-labelledby</code>';
+		$html   = 'attributes <code>role="group"</code> et <code>aria-labelledby</code>';
+		$result = 'attributes <code>role="group"</code> et <code>aria-labelledby</code>';
 
```

Comment:
This is an incorrect change as the sentence is in French.

Reply:
Oh no! I am didn't see "et" in that line.

Reply:
Done 🍏

Reply:
Mhm, somehow I still see the old version here.